development/build/index.js

```diff
@@ -132,6 +132,8 @@ async function defineAndRunBuildTasks() {
         'stateHooks',
         'sentryHooks',
         'sentry',
+        'JSON',
```

Comment:
Do we know where these are being used? Are we sure it's related to Sentry?

Reply:
I'm guessing that they're used by some dependency not related to Sentry, in which case they'd be more appropriate to list under the "globals chrome driver needs to function (test env)" section

Reply:
Not sure where these are being used, so I changed them to the other section as per your suggestion